src/assets/css/index.css

```diff
@@ -119,6 +149,46 @@ a {
     background-color: #EE4266;
 }
 
+.murr-button__success {
+    background-color: #61E294;
+}
+
+.murr-button__success > i {
+    color: white;
+}
+
+.murr-button__success > span {
+    color: white;
+}
+
+.murr-button__success:hover {
+    background-color: #53cb89;
```

Comment:
what do you think about css variables?
https://developer.mozilla.org/ru/docs/Web/CSS/Using_CSS_custom_properties